plugins/diff/erofs/differ_linux.go

```diff
@@ -299,7 +301,8 @@ func (s erofsDiff) Apply(ctx context.Context, desc ocispec.Descriptor, mounts []
 		r: io.TeeReader(processor, digester.Hash()),
 	}
 
-	err = erofsutils.ConvertTarErofs(ctx, rc, layerBlobPath, s.mkfsExtraOpts)
+	u := uuid.NewSHA1(uuid.NameSpaceURL, []byte("oci:/v2/erofs/blobs/"+desc.Digest))
```

Comment:
Doesn't seem relevant to oci

Reply:
Do you have a better naming since it has limited number of userspaces and i think most useful one is url.

Reply:
fixed, PTAL, thanks!